packages/terra-framework-docs/src/terra-dev-site/doc/menu/example/MenuWithCustomIcons.jsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable react/no-access-state-in-setstate */
+import React from 'react';
+import Button from 'terra-button';
+import PropTypes from 'prop-types';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import IconBookmark from 'terra-icon/lib/icon/IconBookmark';
+import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
+import IconEnvelope from 'terra-icon/lib/icon/IconEnvelope';
+import IconPrinter from 'terra-icon/lib/icon/IconPrinter';
+import classNames from 'classnames/bind';
+import styles from './BasicMenu.module.scss';
+import Menu from '../../../../../../terra-menu/lib/Menu';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  isArrowDisplayed: PropTypes.bool,
+  contentWidth: PropTypes.string,
+  boundingRef: PropTypes.func,
+};
+
+class MenuWithCustomIcons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.handleAction = this.handleAction.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.state = {
+      open: false,
+      actionClickCount: 0,
+    };
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleOnClick(event) {
+    event.preventDefault();
+    this.handleRequestClose();
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  handleCloseOnClick(event) {
+    event.preventDefault();
+    this.handleRequestClose();
+  }
+
+  handleAction(event) {
+    event.preventDefault();
+    const newState = this.state;
+    newState.actionClickCount += 1;
+    this.setState(newState);
```

Comment:
```suggestion
    event.preventDefault();
    const { actionClickCount } = this.state;
    actionClickCount += 1;
    this.setState({ actionClickCount });
```
linter disabler added at the top of this file can be removed !!

Reply:
done:
https://github.com/cerner/terra-framework/pull/1592/commits/a6bf9828504b49e233edb281e95de7bee74d7d9e